{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)"},"metadata":{"symbolKind":"case","roleHeading":"Case","modules":[{"name":"PasswordAndNfinityFramework"}],"role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"network"},{"kind":"text","text":"("},{"text":"cause","kind":"externalParam"},{"kind":"text","text":": "},{"text":"CommunicatorError","kind":"typeIdentifier","preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO"},{"kind":"text","text":")"}],"title":"NfinityCloudError.network(cause:)","externalID":"s:27PasswordAndNfinityFramework0C10CloudErrorO7networkyAC012CommunicatorD00hF0O_tcACmF"},"hierarchy":{"paths":[["doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError"]]},"schemaVersion":{"minor":3,"patch":0,"major":0},"kind":"symbol","abstract":[{"type":"text","text":"Case represents network error"}],"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/passwordandnfinityframework\/nfinityclouderror\/network(cause:)"]}],"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["Mac Catalyst"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"network"},{"text":"(","kind":"text"},{"text":"cause","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"CommunicatorError","preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO"},{"text":")","kind":"text"}],"languages":["swift"]}]}],"references":{"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework":{"kind":"symbol","url":"\/documentation\/passwordandnfinityframework","role":"collection","title":"PasswordAndNfinityFramework","abstract":[],"identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework","type":"topic"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError/network(cause:)":{"abstract":[{"type":"text","text":"Case represents network error"}],"title":"NfinityCloudError.network(cause:)","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror\/network(cause:)","role":"symbol","kind":"symbol","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError\/network(cause:)","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"network"},{"kind":"text","text":"("},{"text":"cause","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"CommunicatorError","kind":"typeIdentifier","preciseIdentifier":"s:21CommunicatorFramework0A5ErrorO"},{"kind":"text","text":")"}],"type":"topic"},"doc://com.aheaditec.widgetizing.PasswordAndNfinityFramework/documentation/PasswordAndNfinityFramework/NfinityCloudError":{"abstract":[{"text":"Represents an enum hierarchy for errors related to the NfinityÂ Cloud.","type":"text"}],"title":"NfinityCloudError","url":"\/documentation\/passwordandnfinityframework\/nfinityclouderror","role":"symbol","navigatorTitle":[{"text":"NfinityCloudError","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/com.aheaditec.widgetizing.PasswordAndNfinityFramework\/documentation\/PasswordAndNfinityFramework\/NfinityCloudError","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"NfinityCloudError","kind":"identifier"}],"type":"topic"}}}